pattern-distributed/sp/wso2sp-manager-1-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         pattern: wso2sp-pattern-distributed
     spec:
       containers:
-      - image: docker.wso2.com/wso2sp-manager:4.3.0
+      - image: wso2/wso2sp-manager:4.4.0
         name: wso2sp-manager-1
         imagePullPolicy: Always
```

Comment:
Shall we update the pull policy to IfNotPresent

Reply:
Check all images

Reply:
@ThilinaManamgoda, @DilanUA if consider this as an improvement, we may have to include this as part of the differences which we switch between resources for users with and without a WSO2 subscription.

Reply:
@pcnfernando IMO, we should keep the `imagePullPolicy` as it is. This is since, the underlying Docker image tag of a WSO2 product Docker image may change if a corresponding product Docker resource release is made. Ideally, every time we make a product Docker resource release, we have to update the relevant product Docker image at Docker Hub (though the image tag remains the same).

Reply:
If we are doing a release, anyway the tag will be updated. Are you referring to wum updates. In those case are we creating the docker images with the same tag.